BuildResidentialHPXML/measure.rb

```diff
@@ -866,25 +866,25 @@ def arguments(model)
 
     arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('skylight_area_front', true)
     arg.setDisplayName('Skylights: Front Roof Area')
-    arg.setDescription("The amount of skylight area on the building's front conditioned roof facade.")
+    arg.setDescription("The amount of skylight area on the unit's front conditioned roof facade.")
     arg.setDefaultValue(0)
     args << arg
 
     arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('skylight_area_back', true)
     arg.setDisplayName('Skylights: Back Roof Area')
-    arg.setDescription("The amount of skylight area on the building's back conditioned roof facade.")
+    arg.setDescription("The amount of skylight area on the unit's back conditioned roof facade.")
     arg.setDefaultValue(0)
     args << arg
 
     arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('skylight_area_left', true)
     arg.setDisplayName('Skylights: Left Roof Area')
-    arg.setDescription("The amount of skylight area on the building's left conditioned roof facade.")
+    arg.setDescription("The amount of skylight area on the unit's left conditioned roof facade (when viewed from the front).")
     arg.setDefaultValue(0)
     args << arg
 
     arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('skylight_area_right', true)
     arg.setDisplayName('Skylights: Right Roof Area')
-    arg.setDescription("The amount of skylight area on the building's right conditioned roof facade.")
+    arg.setDescription("The amount of skylight area on the unit's right conditioned roof facade (when viewed from the front).")
```

Comment:
Not related to this PR again, but there are 4 skylight arguments based on orientation. What happens if there is a flat roof (i.e., w/o orientation)?